ModularTegustation/tegu_virology.dm

```diff
@@ -0,0 +1,123 @@
+// Virology "advanced" symptoms.
+/datum/symptom/adv_transmit
+	name = "TCS-TSA 1" // Tiger Co Syndicate - Transmission Speed Accelerator
+	desc = "Some sort of a dark, rapidly developing bacteria; Although it makes presence of the virus very obvious, \
+	it rapidly transmits and evolves to the last stages almost instantly, possibly killing everything in the process."
+	stealth = -9
+	resistance = -5
+	stage_speed = 9
+	transmittable = 7
+	level = 19 // Can't get it normally
+
+/datum/symptom/adv_everything
+	name = "TCS-MPA 2" // Tiger Co Syndicate - Multi-Purpose Accelerator
+	desc = "A multi-purpose strain, capable of increasing most statistics of a virus by a small amount."
+	stealth = 3
+	resistance = 5
+	stage_speed = 5
+	transmittable = 3
+	level = 19
+
+/datum/symptom/adv_stealth
+	name = "TCS-SRA 3" // Tiger Co Syndicate - Stealth Resistance Accelerator
+	desc = "A rare symptom, previously unknown to Nanotrasen. \
+	This symptom makes virus near to impossible to spot, although hinders its ability to develop fast enough."
+	stealth = 7
+	resistance = 5
+	stage_speed = -9
+	transmittable = -5
+	level = 19
+
+/datum/symptom/fake_oxygen // Fake self-respiration, heals until stage 5, then begins to kill.
```

Comment:
stealth = 1
	resistance = -3
	stage_speed = -3
	transmittable = -4

This is what regular self-respiration gives, since this is supposed to be disguised as self-resp, this should have just about the same stats

Reply:
Changed, but it will give increased stealth(2 instead of 1).